docs/build.md

```diff
@@ -28,8 +34,14 @@ $GOPATH/src/github.com/google/cadvisor $ godep go test ./... -test.short
 
 For integration tests, see the [integration testing](integration_testing.md) page.
 
+## Running Built Binary
+
 Now you can run the built binary:
 
 ```
 $GOPATH/src/github.com/google/cadvisor $ sudo ./cadvisor
 ```
+
+If you plan to develop cAdvisor, read about the
+[Kubernetes Github Flow](https://docs.google.com/presentation/d/1HVxKSnvlc2WJJq8b9KCYtact5ZRrzDzkWgKEfm0QO_o/pub?start=false&loop=false&delayms=3000),
```

Comment:
I think we don't need this bit. The usual github help applies to cadvisor. Github bootcamp already covers this.